main.js

```diff
@@ -1,57 +1,75 @@
-const form = document.querySelector('#form');
-const mainHolder = document.querySelector('.container');
-const bookInput = document.querySelector('#book');
+/* eslint-disable  max-classes-per-file */
+const bookList = document.querySelector('#book-list');
+const bookInput = document.querySelector('#title');
 const authorInput = document.querySelector('#author');
 const buttonAdd = document.querySelector('#add');
 
-let books = [];
-
-function show() {
-  const books = JSON.parse(localStorage.getItem('books'));
-  for (let index = 0; index < books.length; index += 1) {
-    bookInput.value = books[index].book;
-    authorInput.value = books[index].author;
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
   }
 }
-form.addEventListener('load', () => {
-  show();
-});
 
-if (localStorage.getItem('books')) {
-  books = JSON.parse(localStorage.getItem('books'));
-}
+class SavelocalItems {
+  static getBook() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
 
-function addBook(book, author) {
-  const newBook = { book, author };
-  books.push(newBook);
-  localStorage.setItem('books', JSON.stringify(books));
+  static saveItem(book) {
+    const books = SavelocalItems.getBook();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBooks(author) {
+    const books = SavelocalItems.getBook();
+    const updatedBooks = books.filter((book) => book.author !== author);
```

Comment:
- [ ] Currently, if I have Book 1 and Book 2 with the same author when I click remove on one of them both of them are getting removed. I highly suggest you generate a random ID and add it to the specific book when creating it. That way it will be easier to remove it using the ID. ✍️